src/WrappedMToken.sol

```diff
@@ -433,24 +449,18 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
     function _claim(address account_, uint128 currentIndex_) internal returns (uint240 yield_) {
         Account storage accountInfo_ = _accounts[account_];
 
-        if (!accountInfo_.isEarning) return 0;
-
-        uint128 index_ = accountInfo_.lastIndex;
-
-        if (currentIndex_ == index_) return 0;
+        if (!_isEarning(accountInfo_)) return 0;
 
         uint240 startingBalance_ = accountInfo_.balance;
 
-        yield_ = _getAccruedYield(startingBalance_, index_, currentIndex_);
-
-        accountInfo_.lastIndex = currentIndex_;
+        // NOTE": Account must be migrated before entering this section.
```

Comment:
```suggestion
        // NOTE: Account must be migrated before entering this section.
```

Reply:
Goo catch, but actually, this is wrong for another reason as well, because the comment doesn't even belong in this PR.